cmd/proxy-server/main.go

```diff
@@ -64,6 +64,7 @@ var (
 	clientPolicyFlag = flag.String("client-policy", "", "OPA policy for outbound client actions (i.e. connecting to sansshell servers). If empty no policy is applied.")
 	clientPolicyFile = flag.String("client-policy-file", "", "Path to a file with a client OPA.  If empty uses --client-policy")
 	hostport         = flag.String("hostport", "localhost:50043", "Where to listen for connections.")
+	debugport        = flag.String("debgport", "localhost:50045", "A separate port for http debug pages.")
```

Comment:
Please add another sentence like: "Set this to \"\" to disable."

Reply:
Done.

Your comment also let me notice that I misspelled the flag name :-)